src/core/wb.js

```diff
@@ -188,7 +188,20 @@ var getUrlParts = function( url ) {
 		},
 
 		getId: function() {
-			return "wb-auto-" + ( seed += 1 );
+			var initialSeed = seed,
+				candidateId;
```

Comment:
Wanted to use ``const`` and ``let`` here... but fell back on ``var`` to lessen the risk of issues in really old browsers.

``getId()`` is called in basic HTML mode and obsolete versions of IE are forced into it...